src/stylesheets/elements/_inputs.scss

```diff
@@ -145,16 +145,16 @@ input[type="radio"] + label::before {
   content: '\a0';
   display: inline-block;
   height: 1.8rem;
-  line-height: 0.8;
+  line-height: 1.8rem;
```

Comment:
What if somebody changes their base font sizes, then wouldn't this no longer be aligned? Could this be written so it accounts for that?